.travis.yml

```diff
@@ -6,10 +6,11 @@ otp_release:
 install:
   - mix local.rebar --force;
   - mix local.hex --force
-  - mix deps.get
+  - mix deps.update --all
```

Comment:
Do we not need a get if we have update, even if it's the first time?

Reply:
Will need to check that Austin, standby

Reply:
deps.get gets the exact version in the mix.exs, deps.update gets and updates to the highest version allowed by the mix.exs